Containerfile

```
@@ -0,0 +1,31 @@
+FROM registry.access.redhat.com/ubi9/ubi as builder
```

Let's get this pinned by digest so that renovate will bump it for you.

```suggestion
FROM registry.access.redhat.com/ubi9/ubi:latest@sha256:7575b6e3cc492f856daf8c43f30692d8f5fcd5b7077806dba4bac436ad0a84e8 as builder
```